api/routes/roomRouter.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from "express";
+import roomController from "../controllers/roomController";
+const router = Router();
+
+router.get("", roomController.getRoomList);
+router.post("/postRoom", roomController.postRoom);
+router.get("/info", roomController.getRoomInfo);
+router.patch("/orderStatus", roomController.patchOrderStatus);
+router.patch("/roomName", roomController.patchRoomName);
+router.post("/postUserCount", roomController.postRoomUser);
+router.delete("/deleteRoomUser", roomController.deleteRoomUser);
+router.patch("/category", roomController.updateCategory);
+export default router;
```

Comment:
```suggestion
const roomRouter = Router();

roomRouter.get("", roomController.getRoomList);
roomRouter.post("/postRoom", roomController.postRoom);
roomRouter.get("/info", roomController.getRoomInfo);
roomRouter.patch("/orderStatus", roomController.patchOrderStatus);
roomRouter.patch("/roomName", roomController.patchRoomName);
roomRouter.post("/postUserCount", roomController.postRoomUser);
roomRouter.delete("/deleteRoomUser", roomController.deleteRoomUser);
roomRouter.patch("/category", roomController.updateCategory);
export default roomRouter;
```

index.ts에서는 roomRouter로 import해오셨는데
여기는 router로 export 하고 계시네요!

Reply:
get post patch delete 순서도 지켜주세요 승수님~~^^

Reply:
머야 외부인 쫒아내